src/main/kotlin/no/nav/familie/ef/sak/brev/FamilieDokumentClient.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.ef.sak.brev
+
+import no.nav.familie.http.client.AbstractPingableRestClient
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestOperations
+import java.net.URI
+
+@Component
+class FamilieDokumentClient(@Value("\${FAMILIE_DOKUMENT_URL}")
+                            private val familieDokumentUrl: String,
+                            @Qualifier("utenAuth")
+                            private val restOperations: RestOperations) : AbstractPingableRestClient(restOperations,
```

Comment:
Hvis vi ikke har en `AbstractHealthIndicator` for klienten så har vi egentlige ikke noe bruk for `AbstractPingableRestClient`
Så tenker vi burde legge til en `HealthIndicator` eller bruke `AbstractRestClient`

Reply:
Takk, fikset ✅